nullaway/build.gradle

```diff
@@ -152,7 +152,7 @@ tasks.register('buildWithNullAway', JavaCompile) {
             sourceSets.main.compileClasspath)
     options.errorprone.enabled = true
     options.errorprone {
-        option("NullAway:AnnotatedPackages", "com.uber,org.checkerframework.nullaway")
+        option("NullAway:AnnotatedPackages", "com.uber,org.checkerframework.nullaway,com.google.common")
```

Comment:
Now that I think of it, let's not add this change here, and instead do it in a separate follow-up PR, just to make things cleaner

Reply:
Makes sense, I have removed it from this branch.